.nvmrc

```diff
@@ -0,0 +1 @@
+14.15.4
```

Comment:
nvmrc is pretty nvm specific, so I recommend against this file. If you want to set a minimum version do >= 14 in engine in package.json

Reply:
My problem was that I can’t get the bcrypt module to compile on latest node, so I wanted to set a known working version 

I can’t see any harm adding it